src/wallet/rpcwallet.cpp

```diff
@@ -5234,6 +5269,7 @@ UniValue sc_sendmany(const UniValue& params, bool fHelp)
         ft.address = address;
         ft.nValue = nAmount;
         ft.scId = scId;
+        ft.mcReturnAddress;
```

Comment:
ft.mcReturnAddress = mcReturnAddress;
Is there any test case that should have detect this bug?